nativeruntime/build.gradle

```diff
@@ -149,33 +136,51 @@ task copyNativeRuntimeFromGithubAction {
           }
         }
       }
-      copy {
-        from zipTree(f)
-        include "librobolectric*"
-        rename { String fileName ->
-          fileName = fileName.replaceFirst("librobolectric.*dylib", "librobolectric-nativeruntime.dylib")
-          fileName = fileName.replaceFirst("librobolectric.*so", "librobolectric-nativeruntime.so")
-          return fileName
-        }
-        def os = "linux"
-        if (artifact.name.contains("-mac")) {
-          os = "mac"
-        }
-        def arch = "x86_64"
-        if (artifact.name.contains("-arm64")) {
-          arch = "aarch64"
-        }
-        into "$buildDir/resources/main/native/${os}/${arch}/"
+    }
+  }
+}
+
+
+def copyNativeRuntimeFromGithubAction() {
+  def checkRunId = System.getenv('NATIVERUNTIME_ACTION_RUN_ID')
+  def downloadDir = new File("$buildDir/robolectric-nativeruntime-artifacts-${checkRunId}")
+  fileTree(downloadDir).visit { FileVisitDetails details -> 
+    println(details)
+    // names << details.file.path 
+    copy {
+      from zipTree(details.file)
+      include "librobolectric*"
+      rename { String fileName ->
+        fileName = fileName.replaceFirst("librobolectric.*dylib", "librobolectric-nativeruntime.dylib")
+        fileName = fileName.replaceFirst("librobolectric.*so", "librobolectric-nativeruntime.so")
+        return fileName
+      }
+      def os = "linux"
+      if (details.name.contains("-mac")) {
+        os = "mac"
       }
+      def arch = "x86_64"
+      if (details.name.contains("-arm64")) {
+        arch = "aarch64"
+      }
+      into "$buildDir/resources/main/native/${os}/${arch}/"
     }
   }
 }
 
+
 processResources {
+  def os = osName()
   if (System.getenv('NATIVERUNTIME_ACTION_RUN_ID')) {
```

Comment:
Will `NATIVERUNTIME_ACTION_RUN_ID` and `SKIP_NATIVERUNTIME_BUILD` be documented at Wiki?

Reply:
Yeah, I can add some docs about this, it is most relevant for releases, when you have to make an uber JAR that contains the shared libraries for the native runtime.

This is also a start to fixing https://github.com/robolectric/robolectric/issues/6803. Currently the SNAPSHOTS build is not working.